portfolio/src/main/webapp/script.js

```diff
@@ -16,21 +16,21 @@
  * Gets comments for the page.
  */
 function getComments() {
-  const commElem = document.getElementById('comments-container');
+  const commElem = document.getElementById("comments-container");
   commElem.innerHTML = "";
   var buttonDiv = document.getElementById("delete");
   buttonDiv.innerHTML = "";
   const number = document.getElementById("number").value;
   fetch("/data?number=" + number).then(response => response.json()).then((comments) => {
-    const commElem = document.getElementById('comments-container');
+    const commElem = document.getElementById("comments-container");
      comments.forEach(c => {
         var textDiv = document.createElement("div");
         textDiv.innerHTML = "<div class='comment-text'>" + c.text + "</div>";
         textDiv.classList.add("box");
         commElem.appendChild(textDiv);
         var infoDiv = document.createElement("div");
         infoDiv.innerHTML = "<span class='comment-name'>" + c.name + " " + "</span>" + 
```

Comment:
Looks like this was introduced in a previous PR, but avoid `innerHTML` in general as it can be an [XSS attack](https://en.wikipedia.org/wiki/Cross-site_scripting) vector. I think gson does sanitization but it's good practice to avoid it. `innerText` is generally safer.

I'd recommend using the DOM API to create these spans (specifically [`document.createElement`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) and then set their innerText, then append the new elements to the div container. I'm happy to provide more guidance on this if you have questions or get stuck.

Reply:
Got it, will change. Making a helper function to make comment elements.

Reply:
Also changing everywhere else I used .innerHTML, and reading up more on XSS. Really interesting stuff.